lms/templates/vert_module.html

```diff
@@ -7,7 +11,7 @@
 <div class="vert-mod">
 % for idx, item in enumerate(items):
   <div class="vert vert-${idx}" data-id="${item['id']}">
-    ${item['content'] | n, unicode}
+    ${HTML(item['content'])}
```

Comment:
If I add a script tag to a vertical module in Studio, it will be executed for students. This may have always been the behavior, but doesn't seem good.

https://unit-title-bookmark.sandbox.edx.org/courses/course-v1:edX+DemoX+Demo_Course/courseware/1414ffd5143b4b508f739b563ab468b7/workflow/